CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
   of the commit log.
 
 ## Unreleased
+* Add breadcrumbs from parent for html documents ([PR #1526] (https://github.com/alphagov/govuk_publishing_components/pull/1526)
```

Comment:
Can this match the PR title please? (you can amend the PR title to match this if you like)

Reply:
I've updated the PR title as I think that's a better description of the change. Originally, the ticket was just to add superbreadcrumbs to html documents, but on investigation it was realised that all html documents needed to pull the breadcrumbs from their parent (including superbreadcrumbs).